invenio_rdm_records/services/permissions.py

```diff
@@ -61,6 +61,9 @@ class RDMRecordPermissionPolicy(RecordPermissionPolicy):
         "object-read": "read_files",
     }
 
+    # permission meant for global curators of the instance
+    # (for now applies to internal notes field only
+    can_manage_internal = [Administration()]
```

Comment:
I believe we should not use `Administration`, there is no global curator, and it is not meant for that. The comment is misleading.
This was discussed in IRL. Why changing it back?